SocketIO/9.2.0/SocketIO.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
```

Comment:
Is this SocketIO client the same as the previously named ios-socket-client?

Reply:
yes, but by some reason, it has been renamed. So, SocketIO and ios-socket-client is the same framework